### PR TITLE
feat: initialize chakra-ui

### PR DESCRIPTION
**Pull request by keithradford**
Initialized Chakra UI and added some preliminary styling to the `Popup` component.

https://chakra-ui.com/

### CODE REVIEW

**Comment by keithradford** [comment]
@mattbardal You might wanna check out this branch and make sure I didn't break anything

File: package.json [hunk 1]
```diff
@@ -30,6 +30,10 @@
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
```
Comment (mattbardal) [inline package.json:32]: Might have to update the typescript dependency to `^4.1.3`. See: https://github.com/chakra-ui/chakra-ui/discussions/3686#discussioncomment-550611

Without the change, two typescript errors occur when running `npm run build`

File: src/popup/Popup.tsx [hunk 2]
```diff
@@ -23,23 +26,19 @@ export default function Popup(): JSX.Element {
...
-      <h3>tt.fm+</h3>
-      <br />
-      Twitch Emotes
-      {emotesButton}
-    </div>
+    <Container w="25em" h="15em" bgColor="#243454" color="#a37b04">
+      <Heading>tt.fm+</Heading>
+      <Divider mb="1em" />
+      <VStack justifyContent="space-between">
+        <LabeledSwitch
```
Comment (mattbardal) [inline src/popup/Popup.tsx:69]: Great replacement of the ternary `<input />` code, much cleaner. However, when you click on the icon to open the popup it causes a warning to appear. (see `chrome://extensions`)

**Review by mattbardal — COMMENTED** [review]
I really like how it looks overall, might have to do some optimizing to prevent exceeding the recommended size limit and fixing some build bugs (and it seems that the emote switch still works perfectly!). 

Some more info about the size limit: [stackoverflow](https://stackoverflow.com/questions/49348365/webpack-4-size-exceeds-the-recommended-limit-244-kib) 

One nitpick I have is the color of the switches, can we make them match the color of the tt.fm+ text at the top?

**Pull request merged**